src/app/plugins/Capistrano/templates/capistrano/tasks/drupal8.rb

```diff
@@ -1,90 +1,118 @@
 # Use version of Drush supplied via Composer
-SSHKit.config.command_map[:drush] = "../vendor/drush/drush/drush"
+SSHKit.config.command_map[:drush] = "#{current_path}/services/drupal/vendor/drush/drush/drush"
 
 # Revert the database when a rollback occurs
 Rake::Task["deploy:rollback_release_path"].enhance do
-  invoke "drupal8:revert_database"
+  invoke "drupal8:db:revert"
 end
 
 # Backup the database when publishing a new release
-Rake::Task["deploy:published"].enhance ["drupal8:dbbackup"]
+Rake::Task["deploy:published"].enhance ["drupal8:db:backup"]
 
 # Copy drush aliases after linking the new release
-Rake::Task["deploy:symlink:release"].enhance ["drush:initialize"]
+# Drush 9 doesn't use global aliases.
```

Comment:
I'm concerned about these (https://github.com/forumone/generator-web-starter/pull/153/files#diff-cc74e0f4f9f89ebdc8ad7059def66b3dR12-R25) changes being Drupal9 specific in the drupal8 task?

Reply:
That's actually for Drush 9 rather than Drupal 9 (think WP-CLI for Drupal). Drush 9 is the minimum version recommended for Drupal 8, so any Drupal 8 work we're doing moving forward should use this version of Drush.

Reply:
@slucero ah, OK, thanks for clarifying.